src/jaxoplanet/experimental/starry/s2fft_rotation.py

```diff
@@ -0,0 +1,170 @@
+import jax.numpy as jnp
```

Comment:
I think it's ok to copy this code, but we should add the LICENSE and an acknowledgement in a comment at the top here.